### PR TITLE
How to win

### PR DESCRIPTION
**Pull request by 9728627**
Basically, I just change the text which will be shown when the user wins the game. 
Whenever you reached 50 points and you will win the game. So I changed the text to:

Congratulations!
You have reached 50 points and win the game!
Please enter your name

100px

### CODE REVIEW

File: src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java [hunk 1]
```diff
@@ -110,8 +110,9 @@ private void gameFinished(boolean won) {
 		Character c = character_type ? Character.SHARK : Character.WHALE;
 		
 		if (won) {
-			message = "Please enter your name \n";
-			playerName = JOptionPane.showInputDialog(fishPanel, message, "CS56");
+			message = "Congratulations! \n You have reached 50 points and win the game!\n Please enter your name \n";
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java]: A couple proofreading nitpicks:
 - Get rid of the spaces next to the newlines - they mess up alignment.
 - It should be "won" not "win".
 - "Please enter your name" should end with a period or a colon.
 - The last newline isn't necessary.

File: src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java [hunk 2]
```diff
@@ -110,8 +110,9 @@ private void gameFinished(boolean won) {
...
+            //JOptionPane.showMessageDialog(null, null, "my title", JOptionPane.INFORMATION_MESSAGE);
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java]: Delete your unused code.

File: src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java [hunk 3]
```diff
@@ -559,6 +547,7 @@ public void HowToPlay() {
 		
 		textLabel.setText("<html><h3>Welcome to Fish Animation !</h3>"
 				+ "<p>Hello! The goal of the game is to eat as many fish as you can. "
+                + "Right now, the target point is 50. Whenever you get 50 or more points, you win! "
```
Comment (xavierholt) [inline src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java]: The first part should be "Right now, the target is 50 points."  Fixing the indentation would be nice too.

**Comment by 9728627** [comment]
Thank you. I have some problem with my git. I can not latest version of master branch. I will figure it out later.

**Pull request merged**